### PR TITLE
fix(console): direct to new created tenant and clear create modal on open

### PR DESCRIPTION
**Pull request by darcyYe**
## Summary

direct to new created tenant and clear create modal on open.

## Testing

UTs and ITs.

## Checklist

- [ ] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [ ] docs

OR

- [x] This PR is not applicable for the checklist

### CODE REVIEW

File: packages/console/src/containers/AppContent/components/Topbar/TenantSelector/index.tsx [hunk 1]
```diff
@@ -107,6 +107,7 @@ function TenantSelector() {
           if (tenant) {
             toast.success(t('tenants.tenant_created', { name: tenant.name }));
             void mutate();
+            window.location.assign(new URL(`/${tenant.id}`, window.location.origin).toString());
```
Comment (wangsijie) [inline packages/console/src/containers/AppContent/components/Topbar/TenantSelector/index.tsx:110]: Can we use `navigate` of `react-router`?

Comment (darcyYe) [reply packages/console/src/containers/AppContent/components/Topbar/TenantSelector/index.tsx:110]: We've set basename for this app and the basename comes with current tenant id. We can not use navigate to switch to another tenant id, its out of the scope.

File: packages/console/src/cloud/pages/Main/TenantLandingPage/TenantLandingPageContent/CreateTenantModal/index.tsx [hunk 2]
```diff
@@ -46,12 +47,20 @@ function CreateTenantModal({ isOpen, onClose }: Props) {
...
     formState: { errors, isSubmitting },
     register,
   } = methods;
 
+  // Should see an unfilled form when modal is opened.
+  useEffect(() => {
+    if (isOpen) {
+      reset();
+    }
+  }, [isOpen, reset]);
```
Comment (charIeszhao) [inline packages/console/src/cloud/pages/Main/TenantLandingPage/TenantLandingPageContent/CreateTenantModal/index.tsx:62]: How about just call `reset()` after submitting the form successfully?

File: packages/console/src/containers/AppContent/components/Topbar/TenantSelector/index.tsx (hunk 1 above)
Comment (charIeszhao) [inline packages/console/src/containers/AppContent/components/Topbar/TenantSelector/index.tsx:110]: Have you checked if the newly selected tenant is the current one? If yes, then we don't need to navigate the page

**Review by charIeszhao — APPROVED** [review]

File: packages/integration-tests/src/tests/ui-cloud/smoke.test.ts [hunk 3]
```diff
@@ -154,36 +154,21 @@ describe('smoke testing for cloud', () => {
       'div[class$=ReactModalPortal] div[class*=card][class$=medium] div[class$=footer] button[type=submit]'
     );
 
-    expect(new URL(page.url()).pathname.endsWith(`${defaultTenantId}/get-started`)).toBeTruthy();
+    expect(new URL(page.url()).pathname.endsWith(`/get-started`)).toBeTruthy();
   });
 
-  it('check current tenant list and switch to new tenant', async () => {
+  it('the current tenant should be new tenant', async () => {
```
Comment (charIeszhao) [inline packages/integration-tests/src/tests/ui-cloud/smoke.test.ts:160]: nit: `it('should navigate to the new tenant')`

**Review by wangsijie — APPROVED** [review]

**Pull request merged**